static/game/hero.js

```diff
@@ -41,21 +41,44 @@ export class Hero extends Creature {
     this.bombHue = 126;
     this.bulletColor = "white";
     this.score = 0;
-    this.setBombDamage(10);
+    this.setBombDamage(18);
 
     // collect powerups when you collide with them
-    this.collideMap.set("PowerUp", entity => {
-      playSound("spacey-snd");
-      /** @type {PowerUp} */ (entity).apply(this);
+    this.collideMap.set("PowerUp", (/** @type {PowerUp} */ entity) => {
+      entity.apply(this);
       for (let i = 0; i < 30; i++) {
-        // TODO move this to the destroy of powerup
         let randColor =
           "hsl(" + Math.floor(Math.random() * 360) + ", 100%, 50%)";
         const spark = new Particle(this.pos, randColor, EffectEnum.spark);
         spark.lineWidth = 15;
         spark.multiplier = 8;
         addParticle(spark);
       }
+      this.addPoints(entity.magnitude * PowerUp.POINTS_FACTOR);
```

Comment:
This results in NaN when the "Static" keyword is removed from POINTS_FACTOR in powerup.js